services/database/index.ts

```diff
@@ -133,6 +133,13 @@ export const getEventIdByHash = async (hash: string): Promise<number> => {
   return event!.id;
 };
 
+export const getEventHash = async (id: number): Promise<string> => {
+  const event = await prisma.event.findFirst({
+    where: { id },
+  });
+  return event!.hash;
```

Comment:
shouldn't we ideally throw an error here if the event is not found in the database?